### DISCUSSION TITLE
Asking about 5v Output on the Board with the Teensy4.1

### DISCUSSION THREAD

**Discussion by silver2row** [grblHAL/core#629] [General]
Hello...I am having trouble communicating a driver now to/from the Teensy4.1 on the board:

https://github.com/grblHAL/iMXRT1062/issues/89

I brought it up in that link listed. For some reason, I cannot get the DM332T drivers to understand the commands from the Teensy4.1 from the board set up to machine.

Um, if anyone has used the DM332T drivers before today, please chime in.

Seth

**Comment by silver2row** [comment]
Also...there is no GND on the drivers for putting GND on the `T41U5XBB` headers for the steppers to the driver.

Seth

P.S. Is there anything I can do with these drivers? Also, can I use the controller without a spindle attached? I have a manual spindle now that does not need the controller for support.

**Reply by terjeio** [reply]
Ground connection is not used when using open-collector signalling, ref. Figure 2 on page 4 in the [datasheet](https://www.omc-stepperonline.com/download/DM332T.pdf).

> Also, can I use the controller without a spindle attached?

Yes.

**Reply by silver2row** [reply]
On page 3 under the header OPTO, I see these ideas:

```
Opto-coupler power supply, and the typical voltage is+5V.
Series connect resistors (at the PUL, DIR, ENA terminals) for current-limiting when +12V or +24V used.
```

Does this mean to connect all three together in series with a resistor?

Seth

P.S. So, I would need extra circuits to use with these drivers. Okay.

**Reply by terjeio** [reply]
> Does this mean to connect all three together in series with a resistor?

Not together and only if you have VCC > 5V. See schematic and box in figure 2 about resistors.
